File: practice.py
display = input("Enter 1 to perform addition & enter 2 to perform table & enter 3 to perform subtraction & enter 4 to perform multiplication & enter 5 to perform division & enter 6 if you want to find squre & enter 7 if you want to find cube: ")

if display == "1":
    num1 = int(input("Enter your first number: "))
    num2 = int(input("Enter your second number: "))
    result = num1 + num2
    print("This is your addition:", result)

elif display == "3":
    num1 = int(input("Enter your first number: "))
    num2 = int(input("Enter your second number: "))
    result = num1-num2
    print("This is your subtraction:", result)
    
elif display == "4":
    num1 = int(input("Enter your first number: "))
    num2 = int(input("Enter your second number: "))
    result = num1*num2
    print("This is your multiplication:", result)
    
elif display == "5":
    num1 = int(input("Enter your first number: "))
    num2 = int(input("Enter your second number: "))
    result = num1/num2
    print("This is your division:", result)
    
elif display == "6":
    num1 = int(input("Enter your first number: "))
    result = num1*num1
    print("This is your square:", result)
    
elif display == "7":
    num1 = int(input("Enter your first number: "))
    result = num1*num1*num1
    print("This is your cube:", result)
    
elif display == "2":
    num1 = int(input("Enter your second number: "))
    result1=num1*1
    result2=num1*2
    result3=num1*3
    result4=num1*4
    result5=num1*5
    result6=num1*6
    result7=num1*7
    result8=num1*8
    result9=num1*9
    result10=num1*10
    print("This is your addition:", result1,result2,result3,result4,result5,result6,result7,result8,result9,result10) 

else:
    print("Invalid input. Please enter 1, 2, or 3,4,5 to perform the corresponding operation.")

File: c/practice.c
#include<stdio.h>
int main(){
    int a,b,c,d,e,f,g,h,i,j,k,l,m;
    printf("Please press 1 if you want to generate a table, press 2 if you want to calculate the square root of a number, and press 3 if you want to find the cube of a number, and press 4 if you want to add two number, and if you want to subtraction of two number then press 5, and if you want to multiply then press 6, and if you want to division then press 7.")+ scanf("%d", &m);
    
    if(m==1){
         printf("give me a number :") + scanf("%d", &a);
    b=a*1;
    c=a*2;
    d=a*3;
    e=a*4;
    f=a*5;
    g=a*6;
    h=a*7;
    i=a*8;
    j=a*9;
    k=a*10;
        printf("table of :%d, %d,%d,%d,%d,%d,%d,%d,%d,%d,%d\n",a,b,c,d,e,f,g,h,i,j,k,l);
    }
    
    if(m==2){
        printf("give me a number :") + scanf("%d", &a);
        b=a*a;
        printf("your square is : %d\n",b);
    }
    
    if(m==3){
        printf("give me a number :") + scanf("%d", &a);
        b=a*a*a;
        printf("your cube is : %d\n",b);
    }
    
    if(m==4){
        printf("give me a number :") + scanf("%d", &a);
        printf("give me a number :") + scanf("%d", &b);
        c=a+b;
        printf("your addition is : %d\n",c);
    }
    
    if(m==5){
        printf("give me a number :") + scanf("%d", &a);
        printf("give me a number :") + scanf("%d", &b);
        c=a-b;
        printf("your subtraction is : %d\n",c);
    }
    
    if(m==6){
        printf("give me a number :") + scanf("%d", &a);
        printf("give me a number :") + scanf("%d", &b);
        c=a*b;
        printf("your multiplication is : %d\n",c);
    }
    
    if(m==7){
        printf("give me a number :") + scanf("%d", &a);
        printf("give me a number :") + scanf("%d", &b);
        c=a/b;
        printf("your division is : %d\n",c);
    }
    
    else{
        printf("thank for your time");
    }
    return 0;
}